lib/apiservers/engine/backends/container.go

```diff
@@ -57,6 +57,32 @@ import (
 	"github.com/vmware/vic/pkg/uid"
 )
 
+// valid filters as of docker commit 49bf474
+var acceptedPsFilterTags = map[string]bool{
```

Comment:
The bool is a feature and not just an empty space...if false then it will be rejected as an invalid filter.